kubernetes/package-kafka/kafkapkginstaller.yml

```diff
@@ -0,0 +1,48 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: kafkapkginstaller
+  namespace: openwhisk
+  labels:
+    name: kafkapkginstaller
+spec:
+  template:
+    metadata:
+      labels:
+        name: kafkapkginstaller
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: kafkapkginstaller
+        imagePullPolicy: IfNotPresent 
+        image: openwhisk/kafkapkginstaller
+        env:
+        - name: "APIHOST"
+          valueFrom:
+            configMapKeyRef:
+              name: packages.kafkaprovider
+              key: whisk_api_host
```

Comment:
You might be able to get whisk_api_host from the existing whisk.ingress configmap? 
```
            valueFrom:
              configMapKeyRef:
                name: whisk.ingress
                key: api_host
```